_posts/en/pages/2016-01-01-meetings.md

```diff
@@ -20,6 +20,8 @@ Current meeting schedule:
 Meeting times are also listed on [this Google calendar][meeting
 calendar].
 
+[Here as ical format][meeting calendar ical] to subscribe via Apple calendar.
```

Comment:
Maybe s/Apple calendar/a calendar app/.

Reply:
Changed it. You are right, ical is a general format not only for Apple calendar.